### ISSUE TITLE
Info text not associated with start button

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk_publishing_components#2083]
This was raised as a question on the cross-government govuk-design-system Slack channel.

## Description of the issue
If you provide information text to a button, the text is rendered underneath the button but not linked to it in any way. This means it won't be read out to screenreader users when they focus on the button, and they could miss important information.

## Actual vs expected behaviour
Current behaviour:

When using the [start now button with info text](https://components.publishing.service.gov.uk/component-guide/button/start_now_button_with_info_text), the following HTML is output:

```html
<a class="gem-c-button govuk-button govuk-button--start" role="button" href="#">Start now...[svg here]</a>
<span class="gem-c-button__info-text">Sometimes you want to explain where a user is going to.</span>
```

This means the information text is completely separate from the button/link.

Expected behaviour: The information text is somehow linked to the button/link, e.g: using aria-labelledby

**Comment by chris-gds** [comment]
Investigating the history of this element, I'm a bit unclear about the reasoning of links that are made to look like buttons.
